packages/devtools_app/lib/src/theme.dart

```diff
@@ -9,20 +9,43 @@ import 'common_widgets.dart';
 import 'config_specific/ide_theme/ide_theme.dart';
 import 'ui/theme.dart';
 
+const DEFAULT_FONT_SIZE = 14.0;
+
 /// Constructs the light or dark theme for the app taking into account
 /// IDE-supplied theming.
 ThemeData themeFor({
   @required bool isDarkTheme,
   @required IdeTheme ideTheme,
+  BuildContext context,
```

Comment:
instead of passing in the BuildContext, let's pass in the theme, since it seems we are only using context to access `Theme.of(context)`